CONFIGURATION.md

```diff
@@ -0,0 +1,174 @@
+| `is_multi_column_measure`   | Indicates if the variable represents a single measure that may have one or more subparts, measured across multiple columns (e.g., subscale scores from a single assessment) | No       | boolean                                     | `false`       | `true`                                                   |
+| `can_have_multiple_columns` | Indicates if the variable can be mapped to multiple columns in the dataset (e.g., for multi-diagnosis or multi-assessment scenarios).                                       | No       | boolean                                     | `false`       | `true`                                                   |
+| `same_as`                   | A URI indicating an equivalent concept in an external vocabulary or ontology.                                                                                               | No       | string \| null                              | `null`        | `"https://medra.org/terms/1234"`                         |
+
+#### Type-Specific Fields
+
+**Categorical Variables** (`data_type: "Categorical"`)
+| Field Name | Description | Required | Type | Example |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------ | --------------- |
+| `terms_file` | Path to the vocabulary JSON file containing allowed standardized term options for variable instances, relative to the configuration's directory | Yes | string | `"sexTerms.json"` |
+
+**Continuous Variables** (`data_type: "Continuous"`)
+| Field Name | Description | Required | Type | Example |
+| ---------- | -------------------------------------- | -------- | ----------------------- | -------------------------- |
+| `formats` | Array of allowed format options for column values | Yes | list of Format objects | See Format Configuration below |
+
+**Other Variables** (`data_type: null`)
+
+Variables with `data_type: null` are typically used for unique observation identifiers (e.g., Participant ID, Session ID) and multi-column measures (e.g., assessment tools). Multi-column measures require a `terms_file` to define the available standardized terms.
+
+| Field Name   | Description                                                                                                                                     | Required | Type   | Example                  |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------ | ------------------------ |
+| `terms_file` | Path to the vocabulary JSON file containing allowed standardized term options for variable instances, relative to the configuration's directory | Yes\*    | string | `"assessmentTerms.json"` |
+
+\*Required only for multi-column measures (`is_multi_column_measure: true`).
```

Comment:
```suggestion
```
I don't think this note is accurate, the two fields shouldn't be dependent. For example, in the Neurobagel config diagnosis is not a multi-column measure, but it requires a `terms_file` because it is categorical. Similarly, in the future we may have multi-column measures that are continuous and thus do not require a `terms_file`